### PR TITLE
Add bootstrap documentation

### PR DESCRIPTION
**Pull request by gaiksaya**
Signed-off-by: Sayali Gaikawad <gaiksaya@amazon.com>

### Description
Add bootstrap documentation. Format numerical bullet points (click on `view file` to see the changes). 
 
### Issues Resolved
[List any issues this PR will resolve]
 
### Check List
- [x] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,18 +33,19 @@ OpenSearch Continuous Integration is an open source CI system for OpenSearch and
 
 ### Dev Deployment 
 1. Setup your local machine to credentials to deploy to the AWS Account
-2. Deploy the ci-config-stackusing the with one of the following (takes ~1 minute to deploy) - 
+1. Deploy the bootstrap stack by running `cdk bootstrap` command that sets up required resources to create the stacks. [More info](https://docs.aws.amazon.com/cdk/latest/guide/bootstrapping.html)
```
Comment (abhinavGupta16) [inline README.md:36]: We need to specify the params here as well, -c. Same goes for destroy as well. I have raised an issue to address this. #45

**Review by abhinavGupta16 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (gaiksaya) [reply README.md:36]: bootstrap does not need a param. For other 2 stacks that can be addressed when someone picks up #45

Comment (peternied) [reply README.md:36]: Lets switch all calls to only `npm run cdk...` variant so its consistant.  Advanced users that have cdk installed globally can do what they desire, but it makes it harder to read with effectively double the options

**Review by peternied — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (abhinavGupta16) [reply README.md:36]: I agree, hence the bug. Currently the check is to throw an exception irrespective of the cdk command. This needs to be tied to deploy only in the bug. Try running `cdk bootstrap` to confirm

Comment (gaiksaya) [reply README.md]: Added the documentation. Bug needs to address to throw the error only when required stack (CI-dev in this case) is deployed.

**Review by abhinavGupta16 — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -33,18 +33,21 @@ OpenSearch Continuous Integration is an open source CI system for OpenSearch and
...
 1. Setup your local machine to credentials to deploy to the AWS Account
-2. Deploy the ci-config-stackusing the with one of the following (takes ~1 minute to deploy) - 
-   1. `npm run cdk deploy CI-Config-Dev` or,
-   2. `cdk deploy CI-Config-Dev`
-3. [Optional](#ssl-configuration) Configure the elements of the config stack for SSL configuration
-4. [Optional](#setup-openid-connect-oidc-via-federate) Configure the elements setting up oidc via federate
-5. Deploy the ci-stack, takes ~10 minutes to deploy (parameter values depend on step 3 and step 4)
+1. Deploy the bootstrap stack by running following command that sets up required resources to create the stacks. [More info](https://docs.aws.amazon.com/cdk/latest/guide/bootstrapping.html):
+   1. `npm run cdk bootstrap -- -c useSsl=false -c runWithOidc=false`
+   1. `cdk boostrap -c useSsl=false -c runWithOidc=false`
```
Comment (peternied) [inline README.md:38]: I meant that we only show the top version of this call for consistency 
```suggestion
   1. `npm run cdk bootstrap -- -c useSsl=false -c runWithOidc=false`
```

Comment (peternied) [reply README.md:38]: *For all commands

Comment (gaiksaya) [reply README.md]: Done

**Review by peternied — APPROVED** [review]

**Pull request merged**